runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -143,6 +144,7 @@
 	private int lastAnimation;
 	private boolean loggedInRace;
 	private boolean widgetHiddenChangedOnPvpWorld;
+	private boolean overloadNMZ;
```

Comment:
What purpose does this serve? We can safely call `removeGameTimer(OVERLOAD)` even if the timer hasn't been added.

Reply:
I added `overloadNMZ` to mimic how the `tzhaarTimer` is used and to also limit the number of calls to `removeGameTimer(OVERLOAD)`. I originally had this 
```
if (!isInNightmareZone())
{
	removeGameTimer(OVERLOAD);
}
```
but it gets triggered frequently outside NMZ by just walking around and loading in new regions.

Reply:
Is it bad for it to be called when walking around? I don't think that's an issue.